samplers/jaegerremote/sampler_remote_options.go

```diff
@@ -46,7 +46,12 @@ func getEnvOptions() ([]Option, []error) {
 	// list of errors which will be logged once logger is set by the user
 	var errs []error
 
-	args := strings.Split(os.Getenv("OTEL_TRACES_SAMPLER_ARG"), ",")
+	rawEnvArgs, ok := os.LookupEnv("OTEL_TRACES_SAMPLER_ARG")
+	if !ok {
+		return nil, nil
+	}
```

Comment:
https://github.com/open-telemetry/opentelemetry-specification/blob/e81d06bd5def98b9004e3959ec9d4563c707a818/specification/configuration/sdk-environment-variables.md#parsing-empty-value

> The SDK MUST interpret an empty value of an environment variable the same way as when the variable is unset.

```suggestion
	rawEnvArgs := os.Getenv("OTEL_TRACES_SAMPLER_ARG")
	if args == "" {
		return nil, nil
	}
```